static/js/echo.js

```diff
@@ -142,6 +142,21 @@ exports.is_slash_command = function (content) {
 
 
 exports.try_deliver_locally = function (message_request) {
+
+    // Even though has: filers can be applied locally,
+    // We dont want to locally echo those has: messages
+    // because:
+    //  1. has:image, has:attachment are backend only syntax.
+    //  2. has:link do not get displayed in current narrow
```

Comment:
`has: link` _might_ not get displayed is the case, I think.

Reply:
I am able to send messages with link when has:link narrow is active.